general/g.region/g.region.md

```diff
@@ -20,10 +20,16 @@ projection. Each region also has associated with it the specific
 east-west and north-south resolutions of its smallest units (rectangular
 units called "cells").
 
+The region extent is common between 2D and 3D maps.
+However, the 2D resolution, used for 2D raster maps, is set separately
+from the 3D resolution, used for 3D raster maps.
+Updating the 2D resolution of the region will not affect its 3D resolution,
+and vice versa.
+
```

Comment:
I think I would prefer this info little later in the text, given most users don't use 3D, this may be unnecessarily confusing. I would put it either towards the end of the section or under a new level 3 heading "3D region".

Reply:
I created a `3D region` section and moved this part there.